From c272f8e941ff4685130b9f4432b2b752d6c95a22 Mon Sep 17 00:00:00 2001
From: Aayushi Arora <ayushiarora681999@gmail.com>
Date: Sun, 14 Jun 2020 00:30:19 +0530
Subject: [PATCH] added task1

---
 task1/client3.py | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/task1/client3.py b/task1/client3.py
index d57389d..56e6ae4 100644
--- a/task1/client3.py
+++ b/task1/client3.py
@@ -3,10 +3,12 @@ import time
 import json
 import random
 
-# Server API URLs
+'''client'''
+'''code'''
+
 QUERY = "http://localhost:8080/query?id={}"
 
-# 500 server request
+
 N = 500
 
 
@@ -29,10 +31,8 @@ def getRatio(price_a, price_b):
     return price_a/price_b
 
 
-# Main
 if __name__ == "__main__":
-
-    # Query the price once every N seconds.
+    # price every n seconds
     for _ in range(N):
         quotes = json.loads(urllib.request.urlopen(
             QUERY.format(random.random())).read())
-- 
2.23.0

